README.de.md

```diff
@@ -0,0 +1,104 @@
+![Dwm](https://raw.githubusercontent.com/antoniosarosi/dotfiles/master/.screenshots/dwm.png)
+
+***Language***
+- [🇪🇸 Español](./README.es.md)
+- [Deutsch](README.de.md)
+- 🇺🇸 English
```

Comment:
You're missing the link to the english version here, put [English] inside [] and use parenthesis for the link (https://github.com/antoniosarosi/dwm)